src/main/resources/config.yml

```diff
@@ -284,6 +284,11 @@ Breaks:
       Message: "&cYou have collected 10 red sand blocks!"
       Name: break_10_sand1
       DisplayName: Red Sand Fan
+  "sand|sand:1":
```

Comment:
I don't think the quotes are required. Let's stay consistent with standalone subcategories and remove them. ^^

Reply:
The indentation makes them implicit.